lib/workers/repository/init/merge.ts

```diff
@@ -173,7 +177,7 @@ export async function mergeRenovateConfig(
 ): Promise<RenovateConfig> {
   let returnConfig = { ...config };
   let repoConfig: RepoFileConfig = {};
-  if (config.requireConfig !== 'ignored') {
+  if (config.repoIsOnboarded && config.requireConfig !== 'ignored') {
```

Comment:
Any comments on this change?

Reply:
Don't we merge renovate config from the onboarding branch?

If that scenario doesn't touch this then are we better to avoid calling this function altogether in cases where repo isn't onboarded?

Reply:
Yes we do. We can bypass this for PRs which are not onboarded by storing the onboardingConfig in cache and use it when we don't clone.

Reply:
Just beware that this would have a huge negative impact if we overlook something and get it wrong